.eslintrc.json

```diff
@@ -0,0 +1,51 @@
+{
+  "root": true,
+  "ignorePatterns": [
+    "projects/**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "parserOptions": {
+        "project": [
+          "tsconfig.json",
+          "e2e/tsconfig.json"
+        ],
+        "createDefaultProgram": true
+      },
+      "extends": [
+        "plugin:@angular-eslint/recommended",
```

Comment:
Does recommended cover all the rules in [tslint.json](https://github.com/ng-select/ng-select/pull/1996/files#diff-9db81b3f77968b3ad243c25f280c4e80dd20b2e5770bf5606ccdb4ecb8af9ce8)?

Reply:
I ran lint based on this config, I got following report

```
/Users/jaddap2/WebstormProjects/ng-select/src/ng-select/lib/ng-dropdown-panel.component.ts
  35:15  error  The selector should start with one of these prefixes: "lib" (https://angular.io/guide/styleguide#style-02-07)  @angular-eslint/component-selector

/Users/jaddap2/WebstormProjects/ng-select/src/ng-select/lib/ng-option.component.ts
  14:15  error  The selector should start with one of these prefixes: "lib" (https://angular.io/guide/styleguide#style-02-07)  @angular-eslint/component-selector

/Users/jaddap2/WebstormProjects/ng-select/src/ng-select/lib/ng-select.component.spec.ts
  4150:7  error  Component class names should end with one of these suffixes: "Component" (https://angular.io/styleguide#style-02-03)  @angular-eslint/component-class-suffix
  4241:7  error  Component class names should end with one of these suffixes: "Component" (https://angular.io/styleguide#style-02-03)  @angular-eslint/component-class-suffix
  4248:7  error  Component class names should end with one of these suffixes: "Component" (https://angular.io/styleguide#style-02-03)  @angular-eslint/component-class-suffix

/Users/jaddap2/WebstormProjects/ng-select/src/ng-select/lib/ng-select.component.ts
   62:15  error    The selector should start with one of these prefixes: "lib" (https://angular.io/guide/styleguide#style-02-07)           @angular-eslint/component-selector
   72:5   error    Use @HostBinding or @HostListener rather than the `host` metadata property (https://angular.io/styleguide#style-06-03)  @angular-eslint/no-host-metadata-property
  258:5   warning  Lifecycle interface 'OnInit' should be implemented for method 'ngOnInit'. (https://angular.io/styleguide#style-09-01)   @angular-eslint/use-lifecycle-interface

/Users/jaddap2/WebstormProjects/ng-select/src/ng-select/lib/ng-templates.directive.ts
   4:24  error  The selector should start with one of these prefixes: "lib" (https://angular.io/guide/styleguide#style-02-08)  @angular-eslint/directive-selector
  18:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  23:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  28:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  33:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  38:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  43:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  48:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  53:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  58:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  63:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector
  68:24  error  The selector should be camelCase (https://angular.io/guide/styleguide#style-02-06)                             @angular-eslint/directive-selector


```
we can the following errors since they are false positive

- The selector should be camelCase
-  The selector should start with one of these prefixes: "lib"

but we may want to resolve this
```
   72:5   error    Use @HostBinding or @HostListener rather than the `host` metadata property (https://angular.io/styleguide#style-06-03)  @angular-eslint/no-host-metadata-property
```

Reply:
Fixed on https://github.com/pavankjadda/ng-select/pull/1